service/utils.go

```diff
@@ -88,24 +88,24 @@ func catalogResource(catalog model.Catalog, apiContext *api.ApiContext, envId st
 	}
 }
 
-func templateDefaultVersion(template model.Template, catalogName string, apiContext *api.ApiContext) (string, string) {
+func templateDefaultVersion(template model.Template, catalogName string, apiContext *api.ApiContext) (string, string, string) {
 	var defaultTemplateVersionId string
 	for _, version := range template.Versions {
 		if version.Version == template.DefaultVersion {
 			defaultTemplateVersionId = generateVersionId(catalogName, template, version)
 			selfLink := apiContext.UrlBuilder.ReferenceByIdLink("templates", defaultTemplateVersionId)
-			return selfLink, defaultTemplateVersionId
+			return selfLink, defaultTemplateVersionId, version.Version
```

Comment:
This case is where `DefaultVersion` comes from `config.yml`, correct?

Reply:
Yup